lib/vagrant/ui.rb

```diff
@@ -75,6 +75,10 @@ def color?
       def machine(type, *data)
         @logger.info("Machine: #{type} #{data.inspect}")
       end
+
```

Comment:
Would you please add a documentation block for this method including a brief explanation of its purpose.